.idea/copyright/GPL.xml

```diff
@@ -0,0 +1,6 @@
+<component name="CopyrightManager">
+  <copyright>
+    <option name="notice" value="Copyright (C) &amp;#36;today.year &lt;Your Name&gt;&#10; &#10;This program is free software: you can redistribute it and/or modify&#10;it under the terms of the GNU General Public License as published by&#10;the Free Software Foundation, either version 3 of the License, or&#10;(at your option) any later version.&#10;&#10;This program is distributed in the hope that it will be useful,&#10;but WITHOUT ANY WARRANTY; without even the implied warranty of&#10;MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the&#10;GNU General Public License for more details.&#10;&#10;You should have received a copy of the GNU General Public License&#10;along with this program.  If not, see &lt;https://www.gnu.org/licenses/&gt;." />
```

Comment:
Does the copyright really need to belong to the (latest?) contributer? Can't we just use YacGroup as owner of the copyright? I dislike private property, even for copyrights.

Reply:
I don't know, if it is possible to use a loose group in the copyright. I will check.

Reply:
I think a loose group is not a legal person. 

I added the original author (extracted from the Git repo) of the file as the person. That may not be fully correct because some of the code was original created by @paetz but than converted to Kotlin by @Fabi2607. During this process the tracking information of some files got lost. It is still possible to find whether there was a original Java file that was created by @paetz or not but it is some effort. If you think you contributed a significant amount of code, just add you to the header with a separate line. We already spent a lot of time in this topic and we should take it not too seriously. 

I far as I understand, before we added the license it was public domain, hence everybody could copy the code, modify or redistribute the code without making a statement that it is from this app. At least on the paper, with GPL this should not happen.